jenkins/plugins.txt

```diff
@@ -54,7 +54,7 @@ git-client:3.6.0
 git-server:1.9
 git:4.6.0
 github-api:1.122
-github-branch-source:2.10.0
+github-branch-source:2.10.1
```

Comment:
this version is broken, there's a fix here that we've been asked to test: https://github.com/jenkinsci/github-branch-source-plugin/pull/395